api.js

```diff
@@ -0,0 +1,78 @@
+// API written in express by charmines#1522
+exports.run = async (client) => {
+	const settings = require("./settings.json")
+	const bodyParser = require("body-parser");
+	const moment = require('moment')
+	const erl = require('express-rate-limit')
+	const express = require('express')
+	let app = express()
```

Comment:
```suggestion
	const app = express()
```

Prefer `const` over `let`.